drone-cd/README.md

```diff
@@ -0,0 +1,117 @@
+# Drone CD
+
+The purpose of this document is to show how to create a CI/CD pipeline with drone
+and Docker.
+
+## Setting up Drone
+
+In order to setup Drone, follow the instructions in [here](../drone-ci/README.md).
+
+Once you are done with it, you should have a running instance of drone in AWS.
+
+## Prerequisites
+
+You must have installed the CLI tool for drone. You can find the instructions in [here](http://readme.drone.io/devs/cli/).
+
+You also need access to an AWS account.
+
+## Setting up a continuous delivery pipeline
+
+Drone is built around a powerful concept: containerized plugins. This enables you
+to easily create a plugin by building a docker image. You can find more information
+about it [here](http://drone-python.readthedocs.io/en/latest/writing_a_plugin.html).
+
+There is no configuration required in the Drone server to make the plugins run. All
+the required configuration is stored per project in a file called .drone.yaml in the
+root of the project:
+```
+build:
+  image: golang
+  commands:
+    - go get
+    - go build
+    - go test
+```
+
+This example, will instruct drone to build a container with the base image golang
+and run the commands specified in the `commands` section.
+
+In this case, we are going to deploy a **new version of our containerized app** to the remote
+host after a set of tests is successfully run.
+
+We are going to have 3 steps:
+- Run the tests
+- Push the container to the AWS Container Registry
+- Deploy the container in a remote machine
+
+The three steps are translated into the following configuration:
+```
+build:
+  image: node
+  commands:
+    - npm install --development
+    - npm test
+publish:
+  ecr:
+    access_key: $$ECR_ACCESS_KEY
+    secret_key: $$ECR_SECRET_KEY
+    region: eu-central-1
+    repo: 1234567890.dkr.ecr.eu-central-1.amazonaws.com/your-repo
+    tag: latest
+    file: Dockerfile
+deploy:
+  ssh:
+    host: your-target-host.yourcompany.com
+    user: ubuntu
+    port: 22
+    commands:
+      - docker pull 1234567890.dkr.ecr.eu-central-1.amazonaws.com/your-repo:latest
+      - docker stop drone-lab
+      - docker rm drone-lab
+      - docker run -d -p 3000:3000 --name drone-lab 711655675495.dkr.ecr.eu-central-1.amazonaws.com/drone-lab:latest
```

Comment:
should `711655675495...` not be the same ecr as above? Any way to make the ecr a variable?